### ISSUE TITLE
Imbalanced Image Classification with Complement Cross Entropy

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#134]
# TL;DR
Proposed a loss function for unbalanced data. Proposed an entropy correction term that combines the inverse ratio of the samples with a coefficient term that increases the confidence of the correct class relative to the confidence of the incorrect answer for each sample, which in longtail CIFAR-10 outperforms the COT(the previous study).
![image](https://user-images.githubusercontent.com/24524018/95159090-203b0000-07d8-11eb-9aef-944cf805bfe7.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/2009.02189

# Submission Dates(yyyy/mm/dd)

# Authors and institutions
Yechan Kim, Younkwan Lee, Moongu Jeon
- School of Electrical Engineering and Computer Science, Gwangju Institute of Science and Technology (GIST), Gwangju, Republic of Korea
- Korea Culture Technology Institute, Gwangju, Republic of Korea

# Methods

# Results

# Comments
